### DISCUSSION TITLE
Conditional options select

### DISCUSSION THREAD

**Discussion by pickett2k** [Budibase/budibase#11088] [Help] [answered]
Hi all, I have a table that holds categories and has a relationship to another table that houses multiple types

in a form I want to be able to select a category and then have the next options select field to only show the types that are within that category but I can’t seem to work out how to sort out the relationships, can anyone assist?

**Comment by melohagan** [comment, marked as answer]
Hey @pickett2k 

You would need to be able to filter relationship picker values, but this work still needs to be done: https://github.com/Budibase/budibase/issues/7810
